helpers/QtiTestSanitizer.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2021 (original work) Open Assessment Technologies SA;
+ */
+
+declare(strict_types=1);
+
+namespace oat\taoQtiTest\helpers;
+
+class QtiTestSanitizer
+{
+    public static function sanitizeContent(string $content): string
+    {
+        $content = preg_replace(
+            '/<script[^>]*>.*<\/script>/is',
+            '',
+            htmlspecialchars_decode($content)
```

Comment:
are you sure on this ?

Reply:
We decided to remove script tags with their content.
We can try to use `strip_tags`, but this will require to add all other tags in `allowed` list.
I don't think it's a good idea to remove tags via regex, because every regex can be broken, and it's difficult to cover all the cases.
Maybe if you have different approach - please, share with me 🙂